README.md

```diff
@@ -27,14 +27,40 @@ Visit the [API Reference](https://api-reference.rebilly.com/) for detailed docum
 
 ## Usage
 To create an instance, you need to provide your secret API key that is available in Rebilly in the [Automations > API Keys](https://app.rebilly.com/api-keys) menu.
+If you want to perform a request to a specific organization in which you have access you need to provide an organization identifier. You could see all organizations memberships in the response from `/v2.1/profile` entry point.
+You can find organization ID in `memberships[0].organization.id` in the next response sample:
+```json
+{
+    "id": "11111111-1111-1111-1111-111111111111",
+    "firstName": "User",
+    "lastName": "Name",
+    "email": "user@domain.com",
+    "memberships": [
+        {
+            "organization": {
+                "id": "11111111-1111-1111-1111-111111111111",
+                "name": "Organization Name",
+                "createdTime": "2019-09-04T03:29:58+00:00"
+            },
+            "user": {
+                "id": "11111111-1111-1111-1111-111111111111",
+                "name": "User Name",
+                "email": "user@domain.com"
+            },
+            ...
```

Comment:
Do we want to put invalid json in an example?

Reply:
I wanted to show that JSON will have more information than in the example. And I think `...` shown that.
Otherwise, I could say about that in the sentence above. I'll fix that.